<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mgmt</class>
 <widget class="QWidget" name="mgmt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="storeBook">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>图书入库</string>
   </property>
  </widget>
  <widget class="QPushButton" name="queryBook">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>图书查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lend">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>借/还书</string>
   </property>
  </widget>
  <widget class="QPushButton" name="licenseMgmt">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>借书证管理</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitSys">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>380</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
